forum/settings.py

```diff
@@ -129,11 +130,13 @@
 
 REST_FRAMEWORK = {
     'DEFAULT_AUTHENTICATION_CLASSES': (
-        'rest_framework.authentication.TokenAuthentication',
+        'authentication.authentication.ExpiringTokenAuthentication',
 
     ),
 }
 
+TOKEN_EXPIRE_TIME = timedelta(days=14)
```

Comment:
I think TOKEN_EXPIRATION_TIME sound better.

Thats better to use nouns as names of variables, not verbs.

Reply:
Renamed